lib/common/output_log.c

```diff
@@ -228,46 +260,38 @@ log_end_list(pcmk__output_t *out) {
 
 G_GNUC_PRINTF(2, 3)
 static int
-log_info(pcmk__output_t *out, const char *format, ...) {
-    private_data_t *priv = NULL;
-    int len = 0;
+log_info(pcmk__output_t *out, const char *format, ...)
+{
     va_list ap;
-    char* buffer = NULL;
+    private_data_t *priv = NULL;
 
     CRM_ASSERT(out != NULL && out->priv != NULL);
     priv = out->priv;
 
+    /* Informational output does not get indented, to separate it from other
```

Comment:
I don't see the indentation that we're contrasting this with, but I haven't looked very hard.

Reply:
list items, mainly